src/init/application/application.ts

```diff
@@ -76,27 +75,27 @@ export function createApplicationGenerator(templateName: string):
 
       this.props = await this.prompt(prompts);
       this.props.elementClassName = this.props.elementName.replace(
-          /(^|-)(\w)/g,
-          (_match: string, _p0: string, p1: string) => p1.toUpperCase());
+        /(^|-)(\w)/g,
```

Comment:
Please revert the whitespace only changes. Our style is 4 space indent on wrapping. Try running `npm run format`, if there are changes in code other than what you touched, put those in another commit.